client/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+    "env": {
+        "test": {
+            "plugins": ["@babel/plugin-transform-modules-commonjs", "transform-es2015-modules-commonjs"]
```

Comment:
why?

Reply:
I downloaded 'react-datepicker' package. And it wasn't compatible with Jest. So I had to transform the module.